relayer/query.go

```diff
@@ -139,7 +139,7 @@ func (c *Chain) QueryClientConsensusState(srcHeight, srcClientConsHeight int64)
 		return clientTypes.NewConsensusStateResponse("notfound", nil, nil, 0), nil
 	}
 
-	var cs exported.ConsensusState
+	var cs clientExported.ConsensusState
```

Comment:
```suggestion
	var cs clientexported.ConsensusState
```
this is how we name it in the SDK